packages/react-native-reanimated/android/src/main/reanimated/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ add_library(
     SHARED
     ${REANIMATED_COMMON_SOURCES}
     ${SOURCES_ANDROID}
+    "${ANDROID_SRC_DIR}/main/cpp/PlatformLogger.cpp"
```

Comment:
This should be moved to worklets.

Reply:
I will do it in the follow-up PR. I don't want to have any non-cpp code separated in this PR.